lib/addons/fluent-bit/index.ts

```diff
@@ -0,0 +1,117 @@
+import * as eks from '@aws-cdk/aws-eks';
+import * as es from '@aws-cdk/aws-elasticsearch';
+import { Effect, PolicyStatement } from '@aws-cdk/aws-iam';
+
+import { Constants } from "..";
+import { ClusterInfo } from "../../spi/types";
+import { Team } from "../../spi/team-contracts";
+import { ClusterAddOn } from "../../spi/addon-contracts"
+import { createNamespace } from "../../utils/namespace";
+
+/**
+ * Configuration options for the FluentBit add-on.
+ */
+export interface FluentBitAddOnProps {
+    /**
+     * The namespace for the FluentBit add-on.
+     * @default `logging`
+     */
+    readonly namespace: string;
+
+    /**
+     * Props for Elasticsearch support.
+     */
+    readonly elasticsearch?: es.DomainAttributes
```

Comment:
What other destinations can we support (if any)? If unknown, may be allowing a free form configuration to be passed?